src/redux/actions/questionsActions/questionListCreator.js

```diff
@@ -0,0 +1,7 @@
+import { GET_QUESTIONS } from "../actionTypes"
+
+export function getQuestions() {
+  return {
+    type: GET_QUESTIONS
```

Comment:
I like that you didn't force a payload to be included here. It looks sharper and doesn't add unnecessary complexity.